tests/API/XCCDF/tailoring/all.sh

```diff
@@ -84,6 +84,55 @@ function test_api_xccdf_tailoring_autonegotiation {
     rm -f $result
 }
 
+function test_api_xccdf_tailoring_simple_include_in_arf {
+    local INPUT=$srcdir/$1
+    local TAILORING=$srcdir/$2
+
+    result=`mktemp`
+    $OSCAP xccdf eval --tailoring-file $TAILORING --results-arf $result $INPUT
+    if [ "$?" != "0" ]; then
+        return 1
+    fi
+
+    assert_exists 1 '/arf:asset-report-collection/arf:report-requests/arf:report-request/arf:content/ds:data-stream-collection/ds:component/Tailoring'
+    rm -f $result
+}
+
+function test_api_xccdf_tailoring_profile_include_in_arf {
+    local INPUT=$srcdir/$1
+    local TAILORING=$srcdir/$2
+
+    result=`mktemp`
+    # "xccdf_com.example.www_profile_customized" customizes "xccdf_com.example.www_profile_baseline1"
+    $OSCAP xccdf eval --tailoring-file $TAILORING --profile "xccdf_com.example.www_profile_customized" --results-arf $result $INPUT || [ "$?" == "2" ]
+
+    component_xpath='/arf:asset-report-collection/arf:report-requests/arf:report-request/arf:content/ds:data-stream-collection/ds:component'
+    assert_exists 3 $component_xpath
+    assert_exists 3 $component_xpath'/@timestamp'
+    assert_exists 1 $component_xpath'/xccdf:Tailoring'
+    assert_exists 1 $component_xpath'/xccdf:Tailoring/xccdf:Profile'
+    assert_exists 1 $component_xpath'/xccdf:Tailoring/xccdf:Profile[@id="xccdf_com.example.www_profile_customized"]'
+    assert_exists 1 $component_xpath'/xccdf:Tailoring/xccdf:Profile[@extends="xccdf_com.example.www_profile_baseline1"]'
+    rm -f $result
+}
+
+function test_api_xccdf_tailoring_profile_generate_fix {
+    local INPUT=$srcdir/$1
+    local TAILORING=$srcdir/$2
+
+    tailoring_result='tailoring_res'
+    fix_result='fix_res'
+    # tailoring profile only with "always fail" rule and generate bash fix
+    $OSCAP xccdf eval --tailoring-file $TAILORING --profile "xccdf_com.example.www_profile_customized" --results-arf $tailoring_result $INPUT || [ "$?" == "2" ]
+    $OSCAP xccdf generate fix --tailoring-id "scap_org.open-scap_cref_tests--API--XCCDF--tailoring--baseline.tailoring.xml_tailoring" --result-id xccdf_org.open-scap_test-result_xccdf-com.example.www_profile_customized --results $fix_result $tailoring_result
```

Comment:
The tailoring ID here is dependent on current working directory, which means this won't work if  you do `cd tests/API/XCCDF/tailoring` and you run `make check` in that directory. You'll get `"scap_org.open-scap_cref_.--baseline.tailoring.xml_tailoring"` as an component reference ID. 

I was thinking how to get the component reference ID.

A first idea that comes to my mind is to use absolute paths for files, eg. `local INPUT=$(realpath $srcdir/$1)`. 
However, then you'll have to build the tailoring ID from the absolute path. That would be a PITA. So instead you can hopefully get the compontent reference using an XPath query, eg `xpath -q -e 'string(//ds:component-ref[contains(@id, "_tailoring")]/@id)' tailoring_res`